### PR TITLE
added handling for the 410 GONE response

### PR DESCRIPTION
**Pull request by whiteshadow-gr**

### CODE REVIEW

File: dataplug-google-calendar/conf/logback.xml [hunk 1]
```diff
@@ -8,7 +8,7 @@
 <configuration>
   <conversionRule conversionWord="coloredLevel" converterClass="play.api.libs.logback.ColoredLevel" />
 
-  <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+  <appender name="STDOUT" class="ch.qos .logback.core.ConsoleAppender">
```
Comment (augustinas) [inline dataplug-google-calendar/conf/logback.xml]: Isn't this the reason why log file loading fails?

File: dataplug/app/com/hubofallthings/dataplug/utils/JwtPhataAuthenticatedAction.scala [hunk 2]
```diff
@@ -36,7 +36,7 @@ class JwtPhataAuthenticatedAction @Inject() (
   val parser = bodyParsers.default
 
   def invokeBlock[A](request: Request[A], block: (JwtPhataAuthenticatedRequest[A]) => Future[Result]): Future[Result] = {
-    request.headers.get("X-Auth-Token")
+    request.headers.get("x-auth-token")
```
Comment (augustinas) [inline dataplug/app/com/hubofallthings/dataplug/utils/JwtPhataAuthenticatedAction.scala]: Why is this change necessary?

Comment (whiteshadow-gr) [reply dataplug/app/com/hubofallthings/dataplug/utils/JwtPhataAuthenticatedAction.scala]: It's not "fixing" anything so it's not necessary at all. I though since we use lowercase, at least in my experience, we should replace them to lowercase as well
